command/status.go

```diff
@@ -45,6 +46,9 @@ Status Options:
   -evals
     Display the evaluations associated with the job.
 
+  -all-allocs
+    Display the allocs that matches the job name.
```

Comment:
`Display all allocations matching the job ID, including those from an older instance of the job.`